R/add_phylopic_tree.R

```diff
@@ -6,26 +6,35 @@
 #' functions.
 #'
 #' @inheritParams add_phylopic_base
-#' @param tree The phylogenetic tree object of class `phylo` on which to add
-#' the silhouette.
-#' @param tip The tip labels against which to add the silhouettes.
-#' If not specified, the names of the `img`, `uuid` or `name` vector are used.
-#' @param relWidth The width of each silhouette relative to the plotting area.
-#' @param padding,relPadding Distance to inset each silhouette from the right
-#' edge of the plotting area,
-#' in the plot coordinate system (`padding`) or
-#' relative to the size of the plotting area (`relPadding`).
-#' Negative values offset to the right.
-#' @param \dots Further arguments to pass to `add_phylopic_base()`.
-#' @author [Martin R. Smith](https://orcid.org/0000-0001-5660-1727) 
-#' (<martin.smith@durham.ac.uk>)
-#' @seealso
-#' For trees plotted using \pkg{ggtree}, see [`geom_phylopic()`].
+#' @param tree The phylogenetic tree object of class `phylo` on which to add the
+#'   silhouette.
+#' @param tip The tip labels against which to add the silhouettes. If not
+#'   specified, the names of the `img`, `uuid` or `name` vector are used.
+#' @param align \code{character}. Should each silhouette be aligned to its
+#'   respective tip (`"tip"`, the default) or to the right-hand side of the
+#'   plotting area (`"plot"`)? If `"tip"` is specified, the silhouette is placed
+#'   at the x coordinate of the respective tip, plus any horizontal padding
+#'   specified by `padding` or `relPadding`. If `"plot"` is specified, the
+#'   silhouette is placed at the right-hand side of the plotting area,
+#'   determined by `par("usr")`, plus any horizontal padding specified by
+#'   `padding` or `relPadding`.
+#' @param width,relWidth The width of each silhouette, in the plot coordinate
+#'   system (`width`) or relative to the size of the plotting area (`relWidth`).
+#'   If "NULL" and `height` is specified, the width is determined by the aspect
+#'   ratio of the original image. One of height and width must be "NULL".
+#' @param padding,relPadding Horizontal padding for each silhouette from it's
```

Comment:
For consistency, specify type (and catch typo)
```suggestion
#' @param padding,relPadding `numeric`. Horizontal padding for each silhouette from its
```